iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ThreeWindingsTransformer.java

```diff
@@ -220,6 +220,7 @@ public interface Leg extends RatioTapChangerHolder, PhaseTapChangerHolder, Flows
          */
         Terminal getTerminal();
 
+
```

Comment:
Can you just remove this space?

Reply:
Done on the last very last push (I have done 2 close push)